client/state/data-layer/http-data.ts

```diff
@@ -208,7 +212,7 @@ export const enhancer = next => ( ...args ) => {
 };
 
 type ResourcePair = [DataId, any];
-type ResponseParser = ( apiData: any ) => ResourcePair[];
+type ResponseParser = ( apiData: any ) => ResourcePair;
```

Comment:
this is the wrong type definition.

the response parser doesn't return a `ResourcePair`, it returns _a list of_ `ResourcePair`s